src/wallet/wallet.cpp

```diff
@@ -914,27 +914,15 @@ CWalletTx* CWallet::AddToWallet(CTransactionRef tx, const CWalletTx::Confirmatio
     CWalletTx& wtx = (*ret.first).second;
     bool fInsertedNew = ret.second;
     bool fUpdated = update_wtx && update_wtx(wtx, fInsertedNew);
+    std::set<COutPoint> candidates;
```

Comment:
is the ordering important? if not we should use unordered set

Reply:
@UdjinM6, is this alright?

Reply:
The ordering is important for `setWalletUTXO`, we rely on it in `CWallet::GetSpendableTXs()`: 
>        // setWalletUTXO is sorted by COutPoint, which means that all UTXOs for the same TX are neighbors
>        // skip entries until we encounter a new TX

https://github.com/dashpay/dash/blob/develop/src/wallet/wallet.cpp#L2534-L2535

Reply:
Keeping it as `std::set` as order is important